test/functional/api/v2/interfaces_controller_test.rb

```diff
@@ -24,6 +24,7 @@ def setup
     show_response = ActiveSupport::JSON.decode(@response.body)
     assert !show_response.empty?
     assert "bmc", show_response["type"]
```

Comment:
This is actually incorrect, it should be `assert_equal "bmc", show_response["type"]` as it's just testing that "bmc" is trueish right now.